apps/coscrad-frontend/src/components/commands/command-panel.tsx

```diff
@@ -51,15 +59,29 @@ export const CommandPanel = ({ actions }: CommandProps) => {
                      * an `ok` button via either this panel or a modal.
                      */
                     onSubmitForm={() => {
-                        console.log(
-                            `You submitted: ${selectedCommand.type} with payload: ${JSON.stringify(
-                                formState
-                            )}`
-                        );
+                        const autoFillFields =
+                            commandContext === INDEX_COMMAND_CONTEXT
+                                ? {}
+                                : {
+                                      compositeIdentifier: commandContext,
```

Comment:
We need a source of truth for this property key.

Reply:
Let's do that during [this story](https://www.pivotaltracker.com/story/show/184094873).